src/index.css

```diff
@@ -1,13 +1,15 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+}
+body {
```

Comment:
Reset some styles. In the majority of cases is great. However, take care with this, please. 👍